src/libsyncengine/requests/exclusiontemplatecache.cpp

```diff
@@ -113,7 +113,28 @@ void ExclusionTemplateCache::updateRegexPatterns() {
             regexPattern += "$"; // End of string
         }
 
-        (void) _regexPatterns.emplace_back(std::regex(regexPattern), exclPattern);
+        // If the NFC and NFD forms of the template are different, both should be excluded.
+        SyncName nfcRegexPattern;
+        SyncName nfdRegexPattern;
+        bool normalizationSucceed =
+                CommonUtility::normalizedSyncName(Str2SyncName(regexPattern), nfcRegexPattern, UnicodeNormalization::NFC) &&
+                CommonUtility::normalizedSyncName(Str2SyncName(regexPattern), nfdRegexPattern, UnicodeNormalization::NFD);
+
+        if (!normalizationSucceed || nfcRegexPattern == nfdRegexPattern) {
+            (void) _regexPatterns.emplace_back(std::regex(regexPattern), exclPattern);
+        } else {
+            ExclusionTemplate exclusionTemplateNfc = exclPattern;
+            SyncName nfcTempl;
+            CommonUtility::normalizedSyncName(Str2SyncName(exclPattern.templ()), nfcTempl, UnicodeNormalization::NFC);
```

Comment:
```suggestion
            (void) CommonUtility::normalizedSyncName(Str2SyncName(exclPattern.templ()), nfcTempl, UnicodeNormalization::NFC);
```
Not mandatory, of course, but it tells the reader that we purposely disregard the return value of the function.

Reply:
Done here: https://github.com/Infomaniak/desktop-kDrive/pull/886/commits/e1993fdfe4980c3cc61384aa2f1a2c8942671b7a